src/libs/mysqlHandler.go

```diff
@@ -521,7 +523,7 @@ func GetWorkloadProcessFileSetMySQL(cfg types.ConfigDB, wpfs types.WorkloadProce
 	var results *sql.Rows
 	var err error
 
-	query := "SELECT policyName,clusterName,namespace,containerName,labels,fromSource,settype,fileset FROM " + WorkloadProcessFileSet_TableName
+	query := "SELECT policyName,clusterName,namespace,podname,labels,fromSource,settype,fileset,createdtime,updatedtime FROM " + WorkloadProcessFileSet_TableName
```

Comment:
Your changes seem to have reverted back my changes. I added containerName in the database yesterday